Build issues with OSS-Fuzz

PR to enable OSS-Fuzz integration is here: https://github.com/google/oss-fuzz/pull/4388

Looks like with -Werror on, there's a failure to build due to implicit conversion.
```
src/float32.c:552:45: error: implicit conversion from 'int' to 'float' changes value from 2147483647 to 2147483648 [-Werror,-Wimplicit-const-int-float-conversion]
        scale = (psf->float_int_mult == 0) ? 1.0 : 0x7FFFFFFF / psf->float_max ;
                                                   ^~~~~~~~~~ ~
1 error generated.
```
and if I build without `--enable-werror` then I get a whole bunch of failures:
```
programs/common.c:163:22: error: no member named 'loudness_value' in 'SF_BROADCAST_INFO_2K'
        REPLACE_IF_NEW_INT (loudness_value) ;
        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
programs/common.c:160:11: note: expanded from macro 'REPLACE_IF_NEW_INT'
                {       binfo.x = round (atof (info->x) * 100.0) ; \
                        ~~~~~ ^
programs/common.c:164:22: error: no member named 'loudness_range' in 'SF_BROADCAST_INFO_2K'
        REPLACE_IF_NEW_INT (loudness_range) ;
        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
programs/common.c:160:11: note: expanded from macro 'REPLACE_IF_NEW_INT'
                {       binfo.x = round (atof (info->x) * 100.0) ; \
                        ~~~~~ ^
programs/common.c:165:22: error: no member named 'max_true_peak_level' in 'SF_BROADCAST_INFO_2K'
        REPLACE_IF_NEW_INT (max_true_peak_level) ;
        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
programs/common.c:160:11: note: expanded from macro 'REPLACE_IF_NEW_INT'
                {       binfo.x = round (atof (info->x) * 100.0) ; \
                        ~~~~~ ^
programs/common.c:166:22: error: no member named 'max_momentary_loudness' in 'SF_BROADCAST_INFO_2K'
        REPLACE_IF_NEW_INT (max_momentary_loudness) ;
        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
programs/common.c:160:11: note: expanded from macro 'REPLACE_IF_NEW_INT'
                {       binfo.x = round (atof (info->x) * 100.0) ; \
                        ~~~~~ ^
programs/common.c:167:22: error: no member named 'max_shortterm_loudness' in 'SF_BROADCAST_INFO_2K'
        REPLACE_IF_NEW_INT (max_shortterm_loudness) ;
        ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
programs/common.c:160:11: note: expanded from macro 'REPLACE_IF_NEW_INT'
                {       binfo.x = round (atof (info->x) * 100.0) ; \
                        ~~~~~ ^
programs/sndfile-info.c:379:59: error: no member named 'loudness_value' in 'SF_BROADCAST_INFO_2K'
                printf ("Loudness value           : %6.2f LUFS\n", bext.loudness_value / 100.0) ;
                                                                   ~~~~ ^
... etc
```

Listing here so they can be tracked.

--- a/ossfuzz/ci_oss.sh
+++ b/ossfuzz/ci_oss.sh
@@ -7,8 +7,10 @@ PROJECT_NAME=libsndfile
 # Clone the oss-fuzz repository
 git clone https://github.com/google/oss-fuzz.git /tmp/ossfuzz
 
-# TODO: Verify that the GITHUB variables below are correct for a PR
+# TODO: Verify that the GITHUB variables below are correct for a PR. Exit
+# before proceeding any further.
 env | grep "GITHUB_"
+exit 0
 
 if [[ ! -d /tmp/ossfuzz/projects/${PROJECT_NAME} ]]
 then

--- a/ossfuzz/ossfuzz.sh
+++ b/ossfuzz/ossfuzz.sh
@@ -21,7 +21,7 @@ apt-get -y install autoconf autogen automake libasound2-dev \
 
 # Compile the fuzzer.
 ./autogen.sh
-./configure --enable-werror --enable-ossfuzzers
+./configure --enable-ossfuzzers
 make V=1
 
 # Copy the fuzzer to the output directory.

